app/src/main/java/com/cornellappdev/scoop/ui/components/post/FirstPage.kt

```diff
@@ -159,16 +159,16 @@ fun TransportationSection(
     proceedEnabled: MutableState<Boolean>,
     modifier: Modifier = Modifier
 ) {
-    val items = listOf("Student driver", "Taxi")
+    val items = listOf("Student Driver", "Shared Taxi")
     val selectedValue = rememberSaveable {
         mutableStateOf(
             postScreenViewModel.ride.type
         )
     }
     val isSelectedItem: (String) -> Boolean = {
         rideTypeToString(selectedValue.value) == when (it) {
-            "Student driver" -> "studentdriver"
-            "Taxi" -> "rideshare"
+            "Student Driver" -> "Student Driver"
+            "Shared Taxi" -> "Shared Taxi"
             else -> "asdf"
         }
     }
```

Comment:
This function is kinda weird, what's it doing exactly? Looks like you'll have to keep manually adding case checks to it whenever you add new ride types. Is there any way to just make this use the `items` list above instead of making you manually add more case checks, which might get kinda annoying?